Nos/Views/DiscoverView.swift

```diff
@@ -110,6 +110,9 @@ struct DiscoverView: View {
                     }
                 }
             }
+            .blur(radius: searchController.isNotFindingResults ? 6 : 0)
```

Comment:
I think the blur and opacity are unnecessary because `notFindingResultsView` there's nothing to blur or make transparent in the results view - it's always going to be empty when we show this message.

Reply:
Removed!